tests/test_data_processing.py

```
@@ -815,456 +828,453 @@ def test_create_batch_dict_span_mask(self, processor):
     @pytest.fixture
     def processor(self, mock_config, mock_tokenizer, mock_words_splitter):
         """Create BiEncoderSpanProcessor instance."""
         from gliner.data_processing import BiEncoderSpanProcessor
-        
+
         labels_tokenizer = Mock()
         labels_tokenizer.unk_token = "[UNK]"
         labels_tokenizer.pad_token = "[PAD]"
-        
+
         def mock_label_tokenize(texts, **kwargs):
```

Ruff (0.15.1)

[warning] Unused function argument: `kwargs`

(ARG001)